extensions/PhabBugz/lib/Constants.pm

```diff
@@ -18,12 +18,14 @@ our @EXPORT = qw(
     PHAB_CONTENT_TYPE
     PHAB_FEED_POLL_SECONDS
     PHAB_USER_POLL_SECONDS
+    PHAB_GROUP_POLL_SECONDS
 );
 
 use constant PHAB_ATTACHMENT_PATTERN => qr/^phabricator-D(\d+)/;
 use constant PHAB_AUTOMATION_USER    => 'phab-bot@bmo.tld';
 use constant PHAB_CONTENT_TYPE       => 'text/x-phabricator-request';
 use constant PHAB_FEED_POLL_SECONDS  => 5;
 use constant PHAB_USER_POLL_SECONDS  => 60;
+use constant PHAB_GROUP_POLL_SECONDS => 300;
```

Comment:
@ckolos -- would you want these to be tweakable with environmental variables?
e.g. change the rate at which the different timers run?

Reply:
easy to add:

```perl
 use constant PHAB_FEED_POLL_SECONDS  => $ENV{PHAB_FEED_POLL} // 5;
 use constant PHAB_USER_POLL_SECONDS  => $ENV{PHAB_USER_POLL} // 60;
+use constant PHAB_GROUP_POLL_SECONDS => $ENV{PHAB_GROUP_POLL} // 300;
```